contracts/starknet/space/space.cairo

```diff
@@ -0,0 +1,134 @@
+%lang starknet
+
+from starkware.starknet.common.syscalls import ( get_caller_address, get_block_number )
+from starkware.cairo.common.cairo_builtins import HashBuiltin
+from starkware.cairo.common.math import assert_lt, assert_le, assert_nn, assert_not_zero
+
+struct Vote:
+    member choice: felt # TODO use Choice enum
```

Comment:
There's no need for this TODO as we don't have enum in Cairo.
What we do instead is that we create constants of the numbers of the enum, and put all these constants under the same namespace.
Therefore, we won't have a choice type and this will remain felt

Reply:
Ok my bad, I thought we could use namespace as `types` but it looks like we can't :) I guess I'm just used to Rust and C :)

Fix: [Remove comments asking to use Choice as a type (impossible)](https://github.com/snapshot-labs/sx-core/pull/47/commits/e120f74a7b050abc3fd24e9055a006d92f0828f2)

Reply:
Do you know if there's a way for us to check if the `choice` argument is one of the `Choice` possible value ? Other than doing some `if .... else if ... else if...` for every possible value ?